client/src/components/main/Search.js

```diff
@@ -94,7 +94,8 @@ class Search extends Component {
   };
 
   renderSectionTitle = section => {
-    return <strong>{section.title}</strong>;
+    console.log(section)
```

Comment:
Did you mean to leave this `console.log` here?

Reply:
most certainly not lol. just pushed to fix.